### DISCUSSION TITLE
Show label colors

### DISCUSSION THREAD

**Discussion by suzuki-shunsuke** [community/community#58723] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

This is a feedback about the recent change of the pull request and issue label.

I found we can't see the color of issue and pull request labels.

e.g. https://github.com/suzuki-shunsuke/tfcmt/pull/70

<img width="351" alt="image" src="https://github.com/community/community/assets/13323303/6814fab8-de94-455a-b343-9e8b8c77f5b6">

This is inconvenient because we can't distinguish the state of the issue and pull request by label color.

For example, we could warn the dangerous change by setting a red label before, but we can't do it now.

<img width="634" alt="image" src="https://github.com/community/community/assets/13323303/36d91910-26e2-42aa-b502-0220f5fb307b">

So I prefer the previous behaviour.

**Comment by MasahikoMitsutake** [comment]
I also encountered same issue.

In my case, changing appearance as following works well to enable label colors.

1. Goto Settings -> Appearance.
2. Change Theme mode from Single Theme to Sync with system.
3. Select your some appearance.

It seems all Light themes at Single Theme are all broken, but themes at Sync with system works well.

**Reply by suzuki-shunsuke** [reply]
Thank you so much!
This is what I want.

**Reply by gnprice** [reply]
Clever, and this fixed it for me, thank you!

One quick further thought: even though selecting "Sync with system", it's possible to make it so that one gets the effect that "Single theme" is intended to have. After selecting "Sync with system", one can choose a light theme for both the "day" and "night" settings, so that you get the same light theme regardless of the system theme setting.

**Reply by PatrickRUG** [reply]
That's a workaround, not a solution. It should work always, also in Single Theme

**Reply by anne-gropler** [reply]
>  After selecting "Sync with system", one can choose a light theme for both the "day" and "night" settings, so that you get the same light theme regardless of the system theme setting.

Thanks, that's a good workaround for me!

**Comment by martinwoodward** [comment, marked as answer]
Hey folks - we are currently reverting the change that made this happen and colors will be back shortly. Sorry about that - the issue is impacting logged in users who have a specific light theme selected.

**Update: 14:45** - Confirmed that this change has been reverted. A bug in an update to the CSS used by the site was causing certain light themes for logged in users to have transparent label colors by mistake. This has now been fixed.  Sorry for the confusion - and thanks for letting us know.
